job-service/src/main/java/com/hpe/caf/services/job/api/DatabaseHelper.java

```diff
@@ -377,7 +377,7 @@ public void deleteJob(final String partitionId, String jobId) throws Exception {
 
         try (
                 Connection conn = DatabaseConnectionProvider.getConnection(appConfig);
-                CallableStatement stmt = conn.prepareCall("{call delete_job(?,?)}")
+                PreparedStatement stmt = conn.prepareCall("call delete_job(?,?)")
```

Comment:
Do we actually need to make these changes to the Java code?

According to https://jdbc.postgresql.org/documentation/head/callproc.html:
> PostgreSQL™ supports two types of stored objects, functions that can return a result value and - starting from v11 - procedures that can perform transaction control. Both types of stored objects are invoked using CallableStatement and the standard JDBC escape call syntax {call storedobject(?)}.

Reply:
![image](https://user-images.githubusercontent.com/1658429/96857081-7d7cb580-1456-11eb-891f-f8aef0859dd1.png)
Here, I tried with a CallableStatement and got that error on those failing tests. 

In the code while testing, I have `CallableStatement stmt = conn.prepareCall("{call delete_job(?,?)}")`